constructs/service-api-function.ts

```diff
@@ -52,10 +55,12 @@ export class ServiceApiFunction extends Construct {
 			awsSdkConnectionReuse: true,
 			entry: definition.path,
 			description: definition.description,
-			memorySize: definition.memorySize ?? 192,
+			memorySize: definition.memorySize ?? defaults?.memorySize ?? 192,
 			reservedConcurrentExecutions: definition.reservedConcurrentExecutions,
 			timeout: definition.timeout
-				? cdk.Duration.seconds(Math.max(30, definition.timeout))
```

Comment:
TIL we max this out to 30 seconds. That's useful to know.

Reply:
And I was the one who wrote this comment 🤦‍♂️ https://github.com/faceteer/cdk/blob/main/handlers/api-handler.ts#L56-L61